client/src/components/Module.jsx

```diff
@@ -30,6 +67,16 @@ Module.propTypes = {
   title: PropTypes.string.isRequired,
   body: PropTypes.string.isRequired,
   attachments: PropTypes.string.isRequired,
-  child: PropTypes.arrayOf.isRequired,
+  child: PropTypes.array.isRequired,
+  link: PropTypes.string.isRequired,
 };
+
+Module.defaultProps = {
```

Comment:
You shouldn't have to have default props if you have required props- you might want to check if your ESLint is enabled, because that should catch some of these errors